<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSlider" name="verticalSlider">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>90</y>
     <width>21</width>
     <height>151</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Light sensor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Alarm system</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deactivate">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Deactivate</string>
   </property>
  </widget>
  <widget class="QLabel" name="alarm_info_on">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Alarm has been activated, to deactivate enter password:</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Remaining time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Alarm status: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Motion sensor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="photo.qrc">:/photo/alarm_off.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>90</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="photo.qrc">:/photo/alarm_on.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Start alarm system</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>71</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="photo.qrc">:/photo/n_motion.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>91</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="photo.qrc">:/photo/y_motion.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="status_light_on">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>290</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>status: ON</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_light_off">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>290</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>status: OFF</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_mot_on">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>status: ON</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_mov_off">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>status: OFF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>340</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Stop alarm system</string>
   </property>
  </widget>
  <widget class="QLabel" name="alarm_info_off">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Alarm is deactivated</string>
   </property>
  </widget>
  <widget class="QLabel" name="bell_on">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="photo.qrc">:/photo/signal_on.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bell_off">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="photo.qrc">:/photo/signal_off.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="photo.qrc"/>
 </resources>
 <connections/>
</ui>
